### PR TITLE
Modify auth tests

### PR DESCRIPTION
**Pull request by felire**
## Summary

I modify the tests to map the actions to types. Doing this the test will be more simples.

### CODE REVIEW

File: generators/app/templates/__tests__/redux/auth/actions.js [hunk 1]
```diff
@@ -1,47 +1,33 @@
-import { actionCreators, actions } from '@redux/auth/actions';
+import {actionCreators, actions} from '@redux/auth/actions';
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/redux/auth/actions.js]: Undo this change. There is a linter problem in the project that will be solved in #114

File: generators/app/templates/__tests__/redux/auth/actions.js [hunk 2]
```diff
@@ -1,47 +1,33 @@
...
 import api from '@config/api';
 import AsyncStorage from '@react-native-community/async-storage';
 
-import { store } from '../store';
+import {store} from '../store';
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/redux/auth/actions.js]: Same

File: generators/app/templates/__tests__/redux/auth/actions.js [hunk 3]
```diff
@@ -1,47 +1,33 @@
-import { actionCreators, actions } from '@redux/auth/actions';
+import {actionCreators, actions} from '@redux/auth/actions';
 import api from '@config/api';
 import AsyncStorage from '@react-native-community/async-storage';
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/redux/auth/actions.js]: Sort imports! Global imports first and then local ones

File: generators/app/templates/__tests__/redux/auth/actions.js [hunk 4]
```diff
@@ -1,47 +1,33 @@
...
 
-import { store } from '../store';
+import {store} from '../store';
+import {mapActionsToTypes} from '../utils';
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/redux/auth/actions.js]: `import { mapActionsToTypes } from '../utils';`

File: generators/app/templates/__tests__/redux/auth/actions.js [hunk 5]
```diff
@@ -1,47 +1,33 @@
...
-    const expectedActions = [
-      { type: actions.LOGIN, target: 'currentUser' },
-      { type: actions.LOGIN_SUCCESS, target: 'currentUser', payload: { sessionToken: 'token' } },
-      {
-        preserveFocus: true,
-        routeName: 'App',
-        type: 'Navigation/JUMP_TO',
-      }
-    ];
+    const expectedActions = [actions.LOGIN, actions.LOGIN_SUCCESS, 'Navigation/JUMP_TO'];
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/redux/auth/actions.js]: Much better! 😄

**Comment by felire** [comment]
Done
